files/image_config/interfaces/interfaces.j2

```diff
@@ -52,10 +52,9 @@ iface eth0 inet dhcp
 # The switch front panel interfaces
 {% for (name, prefix) in INTERFACE %}
 allow-hotplug {{ name }}
-iface {{ name }} {{ 'inet' if prefix | ipv4 else 'inet6' }} static
-    mtu {{ PORT[name]['mtu'] if PORT[name]['mtu'] else 9100 }}
-    address {{ prefix | ip }}
-    netmask {{ prefix | netmask if prefix | ipv4 else prefix | prefixlen }}
+iface {{ name }} {{ 'inet' if prefix | ipv4 else 'inet6' }} manual
```

Comment:
why do we still keep up/down in the interfaces.j2? I thought we are going to remove all of them from the interfaces.j2?